domain/src/test/java/org/oppia/android/domain/spotlight/SpotlightStateControllerTest.kt

```diff
@@ -234,15 +247,15 @@ class SpotlightStateControllerTest {
   fun testRetrieveSpotlightViewState_voiceoverPlayIcon_marked_returnsSpotlightStateSeen() {
     markSpotlightSeen(VOICEOVER_PLAY_ICON)
     val retrieveSpotlightStateProvider =
-      spotlightStateController.retrieveSpotlightViewState(profileId, VOICEOVER_PLAY_ICON)
+      spotlightStateController.retrieveSpotlightViewState(profileId0, VOICEOVER_PLAY_ICON)
     val result = dataProviderTestMonitor.waitForNextSuccessfulResult(retrieveSpotlightStateProvider)
     assertEquals(result, SpotlightViewState.SPOTLIGHT_SEEN)
```

Comment:
Oh I missed this before--this should use Truth not JUnit assertions. Ditto for everywhere else in the test suite.

Reply:
done